controllers/webhooks/workloads/integration/helpers/subnamespace_anchors.go

```diff
@@ -2,27 +2,16 @@ package helpers
 
```

Comment:
This file should probably be renamed, since we're no longer dealing with subnamespace anchors